src/redux/reducers/boardGameReducer.js

```diff
@@ -0,0 +1,20 @@
+export const initialState = {
+    allGamesData: [],
+    currentGame: {}
+}
+
+export const boardGameReducer = (state = initialState, action) => {
+    switch (action.type) {
+        case 'LOAD_GAME_DATA':
+            return {
+                allGamesData: action.payload
```

Comment:
We discussed this briefly last night - is the spread operator required here to destructure state?